### DISCUSSION TITLE
Include a new preprocessor

### DISCUSSION THREAD

**Discussion by krummrey** [Mikubill/sd-webui-controlnet#338] [Ideas]
I've been using [Informative Drawings](https://github.com/carolineec/informative-drawings) with a lot of success. I thinks it yields better results than canny or HED in abstracting images into line drawings. How hard would it be to include it in the list of preprocessors? Seems to work quite well with the canny or sketch model. Probably even better with it's own model.

**Comment by AugmentedRealityCat** [comment]
Wow, this is a pretty good photo-to-linedrawing algorithm.
